sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/NetworkInterceptor.kt

```diff
@@ -0,0 +1,31 @@
+package io.sentry.samples.android
+
+import io.sentry.Sentry
+import io.sentry.SpanStatus
+import java.io.IOException
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class NetworkInterceptor : Interceptor {
+    override fun intercept(chain: Interceptor.Chain): Response {
+
+        val request = chain.request()
```

Comment:
Does this line have side effect or is it just a getter?
If this could potentially be slow we could consider getting a timestamp before calling it and setting the transaction timestamp to that explicitly.

Reply:
its just a getter